test/k4FWCoreTest/src/components/ExampleFunctionalProducerMultiple.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2014-2023 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Gaudi/Property.h"
+#include "GaudiAlg/Producer.h"
+#include "k4FWCore/BaseClass.h"
+
+#include "edm4hep/MCParticleCollection.h"
+#include "edm4hep/SimTrackerHitCollection.h"
+#include "edm4hep/TrackCollection.h"
+#include "edm4hep/TrackerHitCollection.h"
+#include "podio/UserDataCollection.h"
+
+#include <string>
+
+// Which type of collections we are producing
+using Float_t         = podio::UserDataCollection<float>;
+using Particle_t      = edm4hep::MCParticleCollection;
+using SimTrackerHit_t = edm4hep::SimTrackerHitCollection;
+using TrackerHit_t    = edm4hep::TrackerHitCollection;
+using Track_t         = edm4hep::TrackCollection;
```

Comment:
Nitpicking here: These typedefs use `_t`, but the example above doesn't. I think at least in the examples we should have consistency.